app/src/main/java/com/breadwallet/BreadApp.java

```diff
@@ -45,6 +34,8 @@ public class BreadApp extends Application {
     public static long backgroundedTime;
     private static Activity currentActivity;
 
+    public static KeyStoreManager keyStoreManager;
```

Comment:
for now we just use this, maybe in the future we can use dependency injection like `koin`

Reply:
Lets. Feel free to add a spike issue in `litewallet-engineering` for us to study [Koin](https://insert-koin.io/)

Reply:
sure, will add Koin later